src/hal/components/limit3.comp

```diff
@@ -2,94 +2,118 @@ component limit3 "Limit the output signal to fall between min and max, limit its
 pin in float in;
 pin out float out;
 pin in bit load "When TRUE, immediately set \\fBout\\fB to \\fBin\\fR, ignoring maxv and maxa";
-pin in float min_=-1e20;
```

Comment:
I *think* that the old version used min_ and max_ here to avoid problems with some systems that predefine min and max (e.g., as functionlike macros).  Let's arrange to check this against the buildbot before accepting.  If I don't do this within 24 hours, please ping me (It's not 100% convenient right this second for me to do it)

Reply:
I just changed it back to `min_` and `max_`.